tools/aa/commands/InitCommand.java

```diff
@@ -77,14 +77,18 @@ public void run(String[] args) {
           config,
           fileUtils.joinPaths(basePath.get(), BASE_FILENAME));
 
-      // Clone StartupOS repo into head:
-      String startupOsPath = fileUtils.joinPaths(basePath.get(), "head", "startup-os");
-      System.out.println("Cloning StartupOS into " + startupOsPath);
-      Git.cloneRepository()
-          .setURI(startuposRepo.get())
-          .setDirectory(Paths.get(startupOsPath).toFile())
-          .call();
-      System.out.println("Completed Cloning");
+      if (!startuposRepo.get().isEmpty()) {
+        // Clone StartupOS repo into head:
+        String startupOsPath = fileUtils.joinPaths(basePath.get(), "head", "startup-os");
+        System.out.println("Cloning StartupOS into " + startupOsPath);
+        Git.cloneRepository()
+            .setURI(startuposRepo.get())
+            .setDirectory(Paths.get(startupOsPath).toFile())
+            .call();
+        System.out.println("Completed Cloning");
+      } else {
```

Comment:
@oferb FYI this may only work if you supply `--startupos_repo ""` as cmdline arg. That's OK if it was intended but I'm not sure somebody would do it 🙂

Reply:
This is primarily because I didn't know how to test that part :-P